docs/manual/execution-lifecycle/roi-metrics.md

```diff
@@ -0,0 +1,44 @@
+# ROI Metrics Data
+
+::: enterprise
+:::
+
+Automation provides value with each execution of a job.  A job can save time, dollars, and most likely a combination of both.  The ROI (return on investment) Metrics integration will help track the user-defined value of each job execution. **ROI Metrics Data** stores arbitrary key value pairs against your job to help understand your ROI per job execution. The metrics will be stored against the each execution of the job, and accessible via API for further analysis. There is also relevant default execution data to help you further understand your savings.
```

Comment:
The metrics will be stored against _the_ each execution of the job, and accessible via API for further analysis.

Remove "the" before "each"